.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,32 @@
+name: Question
```

Comment:
Same thing here. Should we keep this with the Discussions tab now enabled?

Reply:
We can keep this template and it can be archived if it is decided that questions will be removed from issues.
I think questions may be better as issues as lots of questions get resolved quickly with one answer, rather than having a discussion about it.